src/popup/components/TransactionLabel.vue

```diff
@@ -222,6 +221,12 @@ export default defineComponent({
         };
       }
 
+      if (txType.value === TX_TYPE_MDW.GAAttachTx
+        && externalTxType === TX_TYPE_MDW.PayingForTx
+      ) {
```

Comment:
if you are splitting the condition you should either drop whole condition or nothing